test/clj/rems/api/test_end_to_end.clj

```diff
@@ -607,3 +609,92 @@
           (testing "and it doesn't get rejected"
             (is (= "application.state/rejected" (:application/state (api-call :get (str "/api/applications/" app-3) nil
                                                                               api-key applicant-id))))))))))
+
+(deftest test-bona-fide-bot
+  (let [api-key "42"
+        owner-id "owner"
+        applicant-id "bona-fide-applicant"
+        applicant-attributes {:userid applicant-id
+                              :name "Bona Fide Applicant"
+                              :email "applicant@example.com"}
+        referer-id "bona-fide-referer"
+        referer-attributes {:userid referer-id
+                            :name "Bona Fide Referer"
+                            :email "referer@example.com"
+                            :researcher-status-by "so"}
+        bot-attributes {:userid bona-fide-bot/bot-userid
+                        :email nil
+                        :name "bona fide bot"}]
+    (testing "create users"
+      (api-call :post "/api/users/create" applicant-attributes api-key owner-id)
+      (api-call :post "/api/users/create" referer-attributes api-key owner-id)
+      (api-call :post "/api/users/create" bot-attributes api-key owner-id))
+    (let [resource-id
+          (extract-id (api-call :post "/api/resources/create" {:organization {:organization/id "default"}
```

Comment:
Here I would expect the `resource-id` to start the line and continue with `(extract-id ...)`

Reply:
ah missed a spot, fixed